xm-commons-domainevent/src/main/java/com/icthh/xm/commons/domainevent/config/Column.java

```diff
@@ -0,0 +1,43 @@
+package com.icthh.xm.commons.domainevent.config;
+
+import groovy.transform.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+public class Column {
+
+    private List<String> in;
+    private List<String> notIn;
+    private String equals;
+    private String notEquals;
+    private String match;
```

Comment:
Hm, for me will be good to use Pattern here, but maybe can be skipped for the future